content/source/docs/cloud/users-teams-organizations/api-tokens.html.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: "cloud"
+page_title: "API Tokens - Terraform Cloud"
+sidebar_current: "docs-cloud-users-teams-organizations-api-tokens"
+---
+
+# API Tokens
+
+Terraform Cloud supports three distinct types of API tokens with varying levels of access: user, team, and organization. There are differences in access levels and generation workflows for each of these token types, which are outlined below.
+
+API tokens are displayed only once when they are created, and are obfuscated thereafter. If the token is lost, it must be regenerated. When a token is regenerated, the previous token immediately becomes invalid.
```

Comment:
Heh the only thing I noticed the last time we reviewed this line is that this _isn't_ true for user tokens. You can have multiple tokens and the previous 'one' isn't invalidated when you generate a new one.

Reply:
Ah yeah! It's mentioned in the individual team + org sections anyways so this is also redundant. Removing it.